src/main/kotlin/no/nav/familie/ba/sak/kjerne/vilkårsvurdering/preutfylling/PreutfyllBosattIRiketService.kt

```diff
@@ -193,6 +205,16 @@ class PreutfyllBosattIRiketService(
             }
     }
 
+    private fun lagErBosattISvalbardTidslinje(adresser: Adresser): Tidslinje<Boolean> {
+        val bostedsadresserISvalbardTidslinje = lagTidslinjeForAdresser(adresser.bostedsadresser) { it.erPåSvalbard() }
+        val delteBostederISvalbardTidslinje = lagTidslinjeForAdresser(adresser.delteBosteder) { it.erPåSvalbard() }
```

Comment:
Vi mp bruke oppholdsadresse for å sjekke om personen bor på Svalbard
```suggestion
        val oppholdsadresserPåSvalbardTidslinje = lagTidslinjeForAdresser(adresser.oppholdsadresser) { it.erPåSvalbard() }
```